### PR TITLE
Fix #911 Add Eclipse Che Installer

### LINKED ISSUE #911: Add Eclipse Che installer
Add Eclipse Che to the list of components available for installation assuming we are going to achieve this with CDK add-on. It means Che installer will require CDK and enable CDK add-on during installation.  
- [X]     Create empty installer and add Che into list of components (https://github.com/redhat-developer-tooling/developer-platform-install/pull/913)
- [X]     Implement Che add-on (https://github.com/minishift/minishift-addons/pull/43)
- [x]     Install Che add-on in CDK
- [x]     Implement installer for Che that enables 'che' add on on CDKE
- [ ]     Add link to documentation how to start and use Eclipse Che

### PR DESCRIPTION
**Pull request by dgolovin**
image

### CODE REVIEW

**Comment by dgolovin** [comment]
test this

File: browser/model/che.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  static get KEY() {
+    return 'che';
+  }
+
+  installAfterRequirements(progress, success, failure) {
+    progress.setStatus('Installing');
+    let installer = new Installer(this.keyName, progress, success, failure);
+    return Promise.resolve().then(()=>{
+      return installer.exec('minishift config set memory 5GB && minishift addon enable che', this.createEnvironment());
```
Comment (budhrg) [inline browser/model/che.js]: @LalatenduMohanty looks like we are enabling during installation. I think we need to just set memory when user click to install Che. and point to documentation to apply che.

Comment (budhrg) [reply browser/model/che.js]: /cc @dgolovin

Comment (dgolovin) [reply browser/model/che.js]: @budhrg @LalatenduMohanty Why would we call it Eclipse Che Installer then?

Comment (LalatenduMohanty) [reply browser/model/che.js]: @dgolovin is right. Looks good to me. @budhrg the idea of installer is to make the process super easy for user. So this is the right thing to do as Che would be a separate entity in installer.

**Comment by budhrg** [comment]
Need rebase.

**Pull request merged**